heron/executor/src/python/heron_executor.py

```diff
@@ -466,6 +475,21 @@ def _get_java_instance_cmd(self, instance_info):
       retval[instance_id] = instance_cmd
     return retval
 
+  def _get_jvm_version(self):
+    if not self.jvm_version:
+      cmd = [os.path.join(self.heron_java_home, 'bin/java'),
+             '-cp', self.instance_classpath, 'com.twitter.heron.instance.util.JvmVersion']
+      process = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
Also, bin/java - you might not want to hard code it. You might want to do two path join - we can make it windows proof.